### ISSUE TITLE
Could not resolve com.razorpay:checkout:1.6.+.

### ISSUE THREAD

**Issue by imranhussain546** [razorpay/razorpay-flutter#405]
Please provide all the information requested. Issues that do not follow this format are likely to stall.

## Description
 Could not resolve all files for configuration ':app:debugRuntimeClasspath'.
   > Could not resolve com.razorpay:checkout:1.6.+.
     Required by:
         project :app > project :razorpay_flutter
      > Failed to list versions for com.razorpay:checkout.
         > Unable to load Maven meta-data from https://jcenter.bintray.com/com/razorpay/checkout/maven-metadata.xml.
            > Could not HEAD 'https://jcenter.bintray.com/com/razorpay/checkout/maven-metadata.xml'. Received status code 500 from server: Internal Server Error

## Flutter Version :
 Flutter (Channel stable, 3.22.2, on Microsoft Windows [Version 10.0.19045.4651], locale en-IN)

## Expected Results
Could not resolve all files for configuration ':app:debugRuntimeClasspath'.
   > Could not resolve com.razorpay:checkout:1.6.+.
     Required by:
         project :app > project :razorpay_flutter
      > Failed to list versions for com.razorpay:checkout.
         > Unable to load Maven meta-data from https://jcenter.bintray.com/com/razorpay/checkout/maven-metadata.xml.
            > Could not HEAD 'https://jcenter.bintray.com/com/razorpay/checkout/maven-metadata.xml'. Received status code 500 from server: Internal Server Error

**Comment by wrteam-vimal** [comment]
Same issue here

**Comment by niladri-raychaudhuri** [comment]
Try this
https://github.com/razorpay/razorpay-flutter/issues/400#issuecomment-2285365867

**Comment by vivekshindhe** [comment]
Hey folks, the issue is not with the razorpay-checkout SDK but with mavenCentral. The razorpay-flutter package does not use jcenter. You can verify this [here](https://github.com/razorpay/razorpay-flutter/blob/b/namespace-fix-flutter/android/build.gradle). Also attaching the [link](https://status.maven.org/) to the site which states the outage in OSSRH (mavenCentral) which is why we are seeing this issue. The issue is also different for each because using a specific version has been working for some folks, while it hasn’t for others. You can try to use https://github.com/razorpay/razorpay-flutter/issues/400#issuecomment-2285365867 (Thank you @niladri-raychaudhuri for explaining the steps) fix temporarily to see if that would resolve it for you, while we look into resolving this from our end.

**Comment by vivekshindhe** [comment]
Hey folks, seems like the issue on mavenCentral is resolved. You can check the latest status [here](https://status.maven.org/). Please try to run ./gradlew --refresh-dependencies or ./gradle --refresh-dependencies or run Sync project with Gradle files in Android studio when the android project is opened.

**Comment by wrteam-vimal** [comment]
**Temporary Solution:**

For now, to avoid issues, you should use an older version of the Razorpay Flutter plugin.

Version to Use:

`razorpay_flutter: 1.3.0`

This older version (1.3.0) will work correctly until the current problem is fixed.

**Comment by nimesh-devloper** [comment]
Hey folks,

I think Maven Central has resolved this issue, now just run the **flutter clean** and **flutter pub get** commands in your Terminal and it will work again.

**use the latest razorpay_flutter package**

**Comment by abhijit-hl** [comment]
> Hey folks,
> 
> I think Maven Central has resolved this issue, now just run the **flutter clean** and **flutter pub get** commands in your Terminal and it will work again.
> 
> **use the latest razorpay_flutter package**

Yes, Outage in mavenCentral is resolved now. you can try gradle sync. It is resolved now.
